src/App.css

```diff
@@ -12,3 +17,21 @@
   font-size: calc(10px + 2vmin);
   color: white;
 }
+
+.Canvas {
+  background-color: var(--element-background);
+  position: relative;
+  height: 50vh;
+  width: 50vh;
+}
+
+.Node {
+  height: 2vh;
+  width: 2vh;
+  border-radius: 50%;
+  font-size: 2vh;
```

Comment:
It is a good practice to use rem/em for font size so I'd go with that

Reply:
It is, but here font needs to sized along with the parent element

Reply:
Sounds weird, let's see then ;)